src/utils/storage/index.ts

```diff
@@ -20,6 +20,7 @@ enum StorageKeysEnum {
   LastSelectedMenu = 'lastSelectedMenu',
   BlockedStatus = 'blockedStatus',
   BlockingStatus = 'blockingStatus',
+  IncognitoCreatePostFirstTime = 'incognitoCreatePostFirstTime',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `IncognitoCreatePostFirstTime` to the `StorageKeysEnum` seems appropriate for storing whether the user has created a post in incognito mode for the first time. This could be useful for showing tutorials or prompts related to this feature.
```diff
+   IncognitoCreatePostFirstTime = 'incognitoCreatePostFirstTime',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->